generators/utils.js

```diff
@@ -428,7 +428,7 @@ function getAllJhipsterConfig(generator, force, basePath = '') {
         // merge the blueprint configs if available
         const oldBlueprintName = configuration.blueprint;
         const blueprints = configuration.blueprints || [];
-        if (oldBlueprintName && blueprints.indexOf(oldBlueprintName) === -1) {
+        if (oldBlueprintName && blueprints.findIndex(e => e.name === oldBlueprintName) === -1) {
```

Comment:
FYI this exactly what is done in the other function that deals with merging old and new blueprints info: 
https://github.com/jhipster/generator-jhipster/blob/master/generators/utils.js#L501